samples/react-graph-app-secret-expiration/package.json

```
@@ -9,41 +9,39 @@
     "test": "gulp test"
   },
   "engines": {
-    "node": ">=16.13.0 <17.0.0"
+    "node": ">=16.13.0 <17.0.0 || >=18.17.1 <19.0.0"
   },
   "dependencies": {
-    "@fluentui/react": "^8.61.0",
-    "@microsoft/rush-stack-compiler-3.8": "^0.4.47",
-    "@microsoft/sp-adaptive-card-extension-base": "1.16.1",
-    "@microsoft/sp-core-library": "1.16.1",
-    "@microsoft/sp-http": "1.16.1",
-    "@microsoft/sp-lodash-subset": "1.16.1",
-    "@microsoft/sp-office-ui-fabric-core": "1.16.1",
-    "@microsoft/sp-property-pane": "1.16.1",
-    "@microsoft/sp-webpart-base": "1.16.1",
-    "@pnp/spfx-controls-react": "3.3.0",
+    "@fluentui/react": "8.106.4",
+    "@microsoft/sp-adaptive-card-extension-base": "1.20.0",
+    "@microsoft/sp-core-library": "1.20.0",
+    "@microsoft/sp-http": "1.20.0",
+    "@microsoft/sp-lodash-subset": "1.20.0",
+    "@microsoft/sp-office-ui-fabric-core": "1.20.0",
+    "@microsoft/sp-property-pane": "1.20.0",
+    "@microsoft/sp-webpart-base": "1.20.0",
+    "@pnp/spfx-controls-react": "3.20.0",
     "moment": "^2.29.1",
-    "office-ui-fabric-react": "7.199.1",
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "tslib": "2.3.1"
   },
   "devDependencies": {
-    "@microsoft/eslint-config-spfx": "1.16.1",
-    "@microsoft/eslint-plugin-spfx": "1.16.1",
-    "@microsoft/rush-stack-compiler-3.7": "0.2.3",
-    "@microsoft/rush-stack-compiler-3.9": "0.4.47",
+    "@microsoft/eslint-config-spfx": "1.18.2",
+    "@microsoft/eslint-plugin-spfx": "1.18.2",
     "@microsoft/rush-stack-compiler-4.5": "0.2.2",
-    "@microsoft/sp-build-web": "1.16.1",
-    "@microsoft/sp-module-interfaces": "1.16.1",
+    "@microsoft/rush-stack-compiler-4.7": "0.1.0",
+    "@microsoft/sp-build-web": "1.18.2",
+    "@microsoft/sp-module-interfaces": "1.18.2",
```

if I am not mistaken this should also be `1.20.0`

```suggestion
    "@microsoft/sp-build-web": "1.20.0",
    "@microsoft/sp-module-interfaces": "1.20.0",
```

---

Hi @Adam-it ,

For the following packages:
- @microsoft/eslint-config-spfx
- @microsoft/eslint-plugin-spfx
- @microsoft/sp-build-web
- @microsoft/sp-module-interfaces

I would suggest to keep version 1.18.2.

I'm upgrading this sample from version **1.16.1** and cmdlet `m365 spfx project upgrade` indicated to upgrade those packages to **1.18.2**.
I still tried to upgrade those package to 1.20.0 and `npm install` was failing saying the version does not exist.
`npm install` worked fine with version 1.20.2 (for those 4 packages listed above), but i caused errors when building the web part.

What is your opinion on this?